app/models/campaign.rb

```diff
@@ -80,7 +116,19 @@ class Campaign < ApplicationRecord
   scope :paid_fallback, -> { where paid_fallback: true }
   scope :premium, -> { where(fallback: false).where(paid_fallback: false) }
   scope :job_posting, -> { where job_posting: true }
-  scope :available_on, ->(date) { where(arel_table[:start_date].lteq(date.to_date)).where(arel_table[:end_date].gteq(date.to_date)) }
+
+  scope :starts_on_or_before, ->(date) { where arel_table[:start_date].lteq(date.to_date) }
+  scope :starts_on_or_after, ->(date) { where arel_table[:start_date].gteq(date.to_date) }
```

Comment:
I don't see this being used

Reply:
I vote we keep these scopes even though they aren't being used in the codebase, I'll very likely be using them in a CLI.